### DISCUSSION TITLE
New TensorLy Slack

### DISCUSSION THREAD

**Discussion by JeanKossaifi** [tensorly/tensorly#302] [General]
Hi all!

We now have a Slack for the TensorLy project: 

[![Slack-join](https://img.shields.io/badge/Slack-Join-green)](https://join.slack.com/t/tensorly/shared_invite/zt-wqnts2sk-wbiRX6ml~Xt6~GDYWRPFfg)

Let me know if you have any issue with the invite. 

Happy summer! 🏖️ 🌤️ 🌴 

@aarmey @asmeurer @caglayantuna @cohenjer @juliagusak @MarieRoald @merajhashemi @yngvem

**Comment by OsmanMalik** [comment]
Hi @JeanKossaifi,

It looks like the link no longer works. Is there a more recent link I can use to join the Slack channel?

Best,
Osman

**Comment by aarmey** [comment]
@OsmanMalik here is a link that should never expire: https://join.slack.com/t/tensorly/shared_invite/zt-1rcoppbpz-xkcZ_bqiIGeepJstsIBN8A

**Reply by OsmanMalik** [reply]
Thank you, that worked!

**Reply by JeanKossaifi** [reply]
I've updated the original link, should work too now
